src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/SARIF1019.RuleIdMustBePresentAndConsistent_Invalid.sarif

```diff
@@ -0,0 +1,102 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "SARIF Functional Testing",
+          "rules": [
+            {
+              "id": "SARIF1019",
+              "name": "RuleIdMustBePresentAndConsistent",
+              "shortDescription": {
+                "text": "{0}: The result contains neither result."
+              },
+              "fullDescription": {
+                "text": "{0}: The result contains neither result.ruleId nor result.rule.id."
+              },
```

Comment:
Changing the rule description message will affect this portion of the output file, but you won't have to edit it by hand. Just re-run the test and pick up the new "actual" output.

Reply:
ok!